yumi_moveit_config/config/yumi.srdf

```diff
@@ -69,13 +68,16 @@
     </group>
 
     <!--<group name="left_hand">
```

Comment:
I left it in there to then uncomment for the feature/TwoGrippers branch, but I think you're right and we should remove it so it isn't confusing later on.